src/content/h3/guides/map-making/level-creation/blender-level-modeling/blender-level-creation-additional-info/readme.md

```diff
@@ -166,3 +167,38 @@ There are four aspects to this name so lets break this down.
 * death_barrier - The name for our barrier. This is what the user will give to the `soft_ceiling_enable` script function to disable it.
 
 [wiki-polyhedron]: https://en.wikipedia.org/wiki/Convex_polytope
+
+## Seams
+
+Seams connect BSPs together. Surfaces that are set as a seam surface will connect to another seam surface if the two surfaces are identical (including triangulation) and occupy the same space. The easiest way to do this in Blender is to use the magnetic snapping tool set to edge mode and sliding the surfaces together on an axis. A bsp with seams can only be seamed to a maximum of two other bsps.
+
+![9Ud6igp](https://user-images.githubusercontent.com/45549722/160335249-47e8fca7-d5f8-49b2-b889-8bcbfc928230.png)
+
+A seam can be made by creating a material called `+seam:example_seams_1`
+
+* `+` - Material symbol that lets tool know this is a special material.
+* seam - Tells Tool that this is a seam surface with this material assigned.
+* `:` - The separator between the seam and the name.
+* example_seams_1 - The name for our seam. This is what Tool will use to identify the seams in a bsp.
+
+Once we have multiple bsps that seamed together properly, you can use Tool's `structure` command to import the individual .ASS files for each bsp.
+
+Once all the .ASS files are imported, you can run `structure-seams levels\multi\example_seams` to create the `example_seams.structure_seams` file that will later be referenced in the `scenario` tag.
+
+After the `example_seams.structure_seams` file is created, you might be asked to reimport the .ASS files. This varies depending on what bsps have seams. Reimport those bsps and continue.
+
+Now the `example_seams.structure_seams` file can be referenced in the scenario under the `Structure Bsps` block. 
+
+![ff34mFB](https://user-images.githubusercontent.com/45549722/160336613-d38ef109-3d6d-45c2-8e19-2adae781bed8.png)
```

Comment:
I'm not sure how github feels about hotlinking images, might be best to host these directly on c20.